bucket/filezilla.json

```diff
@@ -15,9 +15,31 @@
     },
     "extract_dir": "FileZilla-3.47.1",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
+        "@'",
+        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>",
+        "<FileZilla3>",
+        "  <Settings>",
+        "    <Setting name=\"Config Location\">config</Setting>",
+        "    <Setting name=\"Disable update check\">1</Setting>",
+        "  </Settings>",
+        "</FileZilla3>",
+        "'@ | Out-File \"$dir\\fzdefaults.xml\" -Encoding Ascii",
+        "",
+        "$oldfzdefault = \"$persist_dir\\fzdefaults.xml\"",
+        "if (Test-Path \"$oldfzdefault\") {",
+        "    Write-Host 'Fixing existing configuration...' -f Yellow",
+        "    $oldConfigDir = \"$((Get-Item \"$persist_dir\").PSDrive.Root)filezilla\"",
```

Comment:
This cannot be correct

My scoop folder is `S:\SCOOP`, filezilla installed in `S:\SCOOP\apps\filezilla`, persist in `S:\SCOOP\persist`

Filezilla configuration from old config is in C:\filezilla

$env:SystemDrive

Reply:
If you customize the scoop directory the configuration is under the corresponding drive. Eg mine is customized to D:\scoop so the old configuration is under D:\filezilla. For you the old filezilla configuration should be under S:\filezilla. So we need to relate to the drive of the persist dir (since it's relative to the old fzdefault.xml which was under persist).
I tested it and it's correct.